src/components/KTable/KTable.vue

```diff
@@ -812,44 +818,37 @@ export default defineComponent({
           text-decoration: underline;
         }
       }
-
-      button,
-      .k-button {
-        margin-top: calc(-1 * var(--KButtonPaddingY, var(--spacing-xs)));
-        margin-bottom: calc(-1 * var(--KButtonPaddingY, var(--spacing-xs)));
-      }
     }
   }
-  // Variants
-  &.has-hover {
-     tbody tr:hover {
-        background-color: var(--KTableHover, var(--blue-100, color(blue-100)));
-      }
-  }
-  &.is-clickable {
-    -webkit-user-select: none;
-    -moz-user-select: none;
-    -ms-user-select: none;
-    user-select: none;
-    tbody tr {
-      cursor: pointer;
-    }
-  }
-  &.side-border {
-    border-collapse: separate;
-    border-spacing: 0 2px;
+}
+</style>
 
-    tbody tr {
-      border-bottom: none;
-    }
+<style lang="scss">
+@import '@/styles/variables';
 
-    tbody tr td:first-child {
-      border-left: 3px solid var(--KTableBorder, var(--steel-200, color(steel-200)));
+.k-table {
+  thead {
+    th {
+      .caret {
+        top: 2px;
+        position: relative;
+        transform: rotate(0deg);
+      }
+
+      &.sortable {
+        &.asc .caret {
+          transform: rotate(-180deg);
+        }
+      }
     }
+  }
 
-    &.has-hover {
-      tbody tr:hover td:first-child {
-        border-left: 3px solid var(--KTableBorder, var(--steel-300, color(steel-300)));
+  tbody {
+    td {
+      button,
+      .k-button {
```

Comment:
You may need to wrap this is a deep selector

```suggestion
      :deep(.k-button) {
```

It's typically needed when you're targeting a class in a child component in Vue 3. 

I didn't check if it's needed for the caret icons above.

Reply:
I'm confused by this. I thought the way to access child components is to use an `unscoped` SCSS block. Do you need `:deep` INSTEAD of `unscoping` or AS WELL AS?

Reply:
TL;DR - if it's not `scoped` then I think it's fine.

I think we want to continue to use scoped where we can instead of unscoped (since this is a component library).

If the block isn't scoped (I didn't check when I wrote this) then it's probably fine, but just verify the rule is actually being applied?

Vue 3 handles scss within child components a little differently if `scoped`: https://vuejs.org/api/sfc-css-features.html#deep-selectors

Reply:
A lot of styles were lost when you originally ported of `KTable` due to the scoping. One of the things that I did in this PR was to split the SCSS into two blocks, one scoped, one unscoped. Only the necessary styles were moved into the unscoped section 👍🏻